Detectors/TRD/base/include/TRDBase/PadCalibrations.h

```diff
@@ -57,33 +51,15 @@ PadCalibrations<T>::PadCalibrations()
   //
   // CalPadStatus constructor
   //
-  //Geometry fgeom;
   int chamberindex = 0;
-  for (auto& roc : mreadOutChamber) { // Range-for!
+  for (auto& roc : mReadoutChamber) { // Range-for!
     roc.init(chamberindex++);
   }
 }
 
-template <class T>
-void PadCalibrations<T>::init()
-{
-  //
-  // CalPadStatus constructor
-  //
-  int chamberindex = 0;
-  for (auto& roc : mreadOutChamber) { // Range-for!
-    roc.init(chamberindex++);
-  }
-}
-
-template <class T>
-void PadCalibrations<T>::reset(int roc, int col, int row, std::vector<T>& data)
-{
-  //reset the readoutchamber
-  //primarily here for setting the values incoming from run2 ocdb, but might find other use cases.
-  // you need to send it the roc as it is used to calculate internal parameters.
-  mreadOutChamber[roc].reset(roc, col, row, data); // it *should* not actually matter as this *should* be set correctly via init.
-}
+// instantiate templates always needed
+template class PadCalibrations<float>; // for LocalT0, LocalVDrift, LocalGainFactor, PadNoise
```

Comment:
I don't think we will have local, i.e. pad-by-pad, T0 or local Vdrift calibrations.

Reply:
Probably not, but the template instantiation will be needed anyhow for the pad-wise gain factor (`LocalGainFactor`)

Reply:
@tdietel I checked and at the moment the values we are using for LocalT0 and LocalVDrift are 1 and 0 resp., so they don't have any effect. So I would indeed remove them now completely. Do you agree?